src/main/java/minegame159/meteorclient/systems/modules/movement/Scaffold.java

```diff
@@ -78,6 +78,20 @@
             .build()
     );
 
+    private final Setting<Boolean> filtFall = sgGeneral.add(new BoolSetting.Builder()
+            .name("Filter Falling Blocks")
```

Comment:
Change this to `"filter-falling-blocks"`
Names with spaces will cause issues with `.setting` commands